apps/component-date/index.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const summary = require('hof-behaviour-summary-page');
+
+module.exports = {
+  name: 'component-date',
+  baseUrl: '/component-date',
```

Comment:
I tried to remove one or the other and unfortunately it does need both items even if they have the same name, do you want to change that in hof?